bus-board/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import React, {useState} from 'react';
+import {getDataForStopPoints} from './apiClients/BusStopPointAPI'
+import {IJsonBus} from "./modules/IJsonBus";
+import {BusDataTable} from "./components/BusDataTable";
+
+async function getBuses(stopPoint: string): Promise<IJsonBus[]> {
+    const bussesArray: IJsonBus[] | undefined = await getDataForStopPoints(stopPoint);
+    return bussesArray != undefined ? bussesArray : new Array<IJsonBus>();
+}
+
+function App(): React.ReactElement {
```

Comment:
Let's be consistent and use the structure I sent on slack for all our components:
const App: React.FC = () => {}